evap/staff/tests/test_tools.py

```diff
@@ -219,6 +222,52 @@ def test_do_nothing_if_test_run(self):
         self.assertEqual(len(messages), 4)
 
 
+class RemoveUserDueToInactivity(TestCase):
+    def test_remove_user_due_to_inactivity(self):
+        inactive_user = baker.make(UserProfile, last_name="Inactive User")
```

Comment:
We try to keep the tests as minimal/simple as possible for the required/targeted assertiveness. I think the last_name here is not relevant to the test, so we could remove it. The library we use, [model_bakery](https://model-bakery.readthedocs.io/en/latest/), should fill in any required fields with random values.

(This is also the case below for last_name in the other tests, and possibly also for some other field values)